pkg/operator/starter.go

```diff
@@ -298,6 +299,17 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		return err
 	}
 
+	invalidWebhookCertsController := metricscontroller.NewMetricsController(
+		"InvalidWebhookCertsController",
```

Comment:
TODO(sur) this does not only observe webhooks but also aggregated API servers. we should rename this.

Reply:
fixed